underfs/oss/src/main/java/tachyon/underfs/oss/OSSUnderFileSystem.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.underfs.oss;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.List;
+
+//import org.slf4j.Logger;
+//import org.slf4j.LoggerFactory;
+
+//import tachyon.Constants;
+
+import tachyon.conf.TachyonConf;
+import tachyon.underfs.UnderFileSystem;
+
+/**
+ * Under file system implementation for Aliyun OSS using the oss SDK library.
+ */
+public class OSSUnderFileSystem extends UnderFileSystem {
+  //private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
+
+  /** Suffix for an empty file to flag it as a directory */
+  //private static final String FOLDER_SUFFIX = "_$folder$";
+  /** Value used to indicate folder structure in S3 */
+  //private static final String PATH_SEPARATOR = "/";
+  /** Aliyun OSS client */
+  // TODO(luoli523523) include OSS dependency late
+  //private OSSClient mClient;
+  /** Bucket name of user's configured Tachyon bucket */
+  //private final String mBucketName;
+  /** Prefix of the bucket, for example s3n://my-bucket-name/ */
+  //private final String mBucketPrefix;
+
+  protected OSSUnderFileSystem(String bucketName, TachyonConf tachyonConf) {
+    super(tachyonConf);
+    // TODO(luoli523523) Auto-generated constructor stub
+  }
+
+  @Override
+  public UnderFSType getUnderFSType() {
+    return UnderFSType.OSS;
+  }
+
+  @Override
+  public void connectFromMaster(TachyonConf conf, String hostname) throws IOException {
+    // TODO(luoli523) Auto-generated method stub
+  }
+
+  @Override
+  public void connectFromWorker(TachyonConf conf, String hostname) throws IOException {
+    // TODO(luoli523) Auto-generated method stub
+  }
+
+  @Override
+  public void close() throws IOException {
+    // TODO(luoli523) Auto-generated method stub
+  }
+
+  @Override
+  public OutputStream create(String path) throws IOException {
+    // TODO(luoli523) Auto-generated method stub
+    return null;
+  }
+
+  @Override
+  public OutputStream create(String path, int blockSizeByte) throws IOException {
+    // TODO(luoli523) Auto-generated method stub
```

Comment:
I am fine with either approach, as long as we specify in this PR description that there is a followup PR (if that is @luoli523 's plan).